src/Security/UserCheckerAuthLogin.php

```diff
@@ -6,23 +6,30 @@
 use Symfony\Component\Security\Core\User\UserInterface;
 use Symfony\Component\Security\Core\Exception\CustomUserMessageAccountStatusException;
 use App\Entity\User;
+use Symfony\Contracts\Translation\TranslatorInterface;
 
 class UserCheckerAuthLogin implements UserCheckerInterface {
 
+    private TranslatorInterface $translator;
+
+    public function __construct(TranslatorInterface $translator) {
```

Comment:
same thing, alter this plz 
```suggestion
    public function __construct(private TranslatorInterface $translator) {
```

Reply:
resolved by @Jonatas-Soares-Alves